Unit Tests/Helpers/XCTest+Networking.swift

```diff
@@ -0,0 +1,73 @@
+import XCTest
+import Mockingjay
+import PusherPlatform
+@testable import PusherChatkit
+
+// MARK: - Networking
+
+internal extension XCTest {
+    
+    struct Networking {
+        
+        // MARK: - Properties
+        
+        static let testInstanceLocator = "test:instance:locator"
+        
+        // MARK: - Paths
+        
+        enum Path: String {
+            
+            case users
+            
+        }
+        
+    }
+    
+}
+
+// MARK: - Mockingjay
+
+internal extension XCTest {
+    
+    // MARK: - Properties
+    
+    private static let instanceLocatorSeparator: Character = ":"
+    
+    // MARK: - Internal methods
+    
+    @discardableResult func stubSubscription(of service: ServiceName, version: ServiceVersion, instanceLocator: String, path: Networking.Path, with jsonFilename: String) -> Mockingjay.Stub? {
+        let bundle = Bundle(for: type(of: self))
```

Comment:
Unfortunately, we cannot. Currently `Bundle.current` is hardcoded to `PusherChatkit` bundle and here we need to get `Unit Tests` bundle. I initially wanted to get rid of the hardcoded value and make this dynamic. It turns out that such thing (relatively easy to do in Objective-C) would require >1000 lines of Swift code and even than it would not return the expected value in 100% situations.